src/components/SingleArticle.jsx

```diff
@@ -29,8 +30,11 @@ function SingleArticle() {
         setArticleVotes((currentArticleVotes=>{
             return currentArticleVotes + 1;
         }))
-        patchArticleById(article_id, 1).catch((err)=>{
-            setApiError(err)
+        patchArticleById(article_id, 'a').catch((err)=>{
+            
```

Comment:
Your voting system currently allows user to Upvote an Article multiple times. The button isn't disabled between votes, allowing users to inflate votes by spam-clicking the button - this is not false per se, but just something for you to consider :)